src/remoted/secure.c

```diff
@@ -74,7 +74,7 @@ void HandleSecure()
     int sock_client;
     int n_events = 0;
     char buffer[OS_MAXSTR + 1];
-    ssize_t recv_b;
+    int recv_b;
```

Comment:
why is there an underflow if ssize_t are signed?

can we have a problem with the recvfrom? I understand no if the int is 4 bytes, and I don't think we compile with a 2 bytes int in the manager side.

Reply:
This change comes from commit https://github.com/wazuh/wazuh/commit/be490d17d8642bd60dde7d68b3913815412ea5a8, which replaces `ssize_t` with `int`.

The reason is that the range of `ssize_t` is [ -1, `SSIZE_MAX` 2³¹-1 ], (typically [ -1, 2³¹-1 ]) while `int` is [ -2³¹, 2³¹-1 ].

`recv_b` is set by `recvfrom`, which is expected to return a value in range [ -1, `OS_MAXSTR` ] ([ -1, 65536 ]), but `nb_recv` may return [ -2, `OS_MAXSTR` ]. `ssize_t` does not guarantee to store `-2`.